libraries/src/HTML/Helpers/Dropdown.php

```diff
@@ -90,7 +90,7 @@ public static function init()
     public static function start()
     {
         // Only start once
-        if (isset(static::$loaded[__METHOD__]) && static::$loaded[__METHOD__] == true) {
+        if (isset(static::$loaded[__METHOD__]) && static::$loaded[__METHOD__]) {
```

Comment:
Could be simplified to:
```suggestion
        if (isset(static::$loaded[__METHOD__])) {
```